client/state/stats/lists/utils.js

```diff
@@ -252,6 +265,39 @@ export const normalizers = {
 	},
 
 	/**
+	 * Returns a normalized statsFollowers object
+	 *
+	 * @param  {Object} data    Stats data
+	 * @return {?Object}         Normalized stats data
+	 */
+	statsFollowers( data ) {
+		if ( ! data ) {
+			return null;
+		}
+		const { total_wpcom, total_email } = data;
+		const subscriberData = get( data, [ 'subscribers' ], [] );
+
+		const subscribers = subscriberData.map( ( item ) => {
+			return {
+				label: item.label,
+				iconClassName: 'avatar-user',
+				icon: parseAvatar( item.avatar ),
+				link: item.url,
+				value: {
+					type: 'relative-date',
+					value: item.date_subscribed
+				},
+				actions: [ {
+					type: 'follow',
+					data: item.follow_data ? item.follow_data.params : false
+				} ]
```

Comment:
In the context of the "Followers", I still think we might want to keep it around.  The follow links shown in the "Followers" module do get usage.  But I do think we should open a PR to remove them to discuss further.